gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -106,9 +119,11 @@ typedef struct _GstTensorFilterProperties
 
   int input_configured; /**< TRUE if input tensor is configured. Use int instead of gboolean because this is refered by custom plugins. */
   GstTensorsInfo input_meta; /**< configured input tensor info */
+  tensors_layout input_layout; /**< data layout info for the input */
```

Comment:
We need to be more specific.
Is this the data layout that the src element (upstream) has given? or the data layout that this filter requires for this input? I guess it is the former, but it might be not that trivial for users. We also need to state that when we need this info and how we will react if this is not given.

Reply:
In the current implementation, it's the layout the filter requires for this input and is provided by the user. The data layout that the src element (upstream) has given will be used once tensor_layout becomes a part of the element capabilities.
More info about the layout has been added.